### PR TITLE
[multi-packages] Migrate plus plugins to use flutter_lints

### LINKED ISSUE #254: Migrate plus plugins to use flutter_lints
https://github.com/fluttercommunity/plus_plugins/pull/519

We need to update `analysis_options_plus.yaml` at the root of our repo.

### PR DESCRIPTION
**Pull request by HakkyuKim**
Closes #254.

References: 
- https://github.com/fluttercommunity/plus_plugins/pull/519
- https://github.com/fluttercommunity/plus_plugins/pull/537

### CODE REVIEW

**Comment by HakkyuKim** [comment]
`webview_flutter` deprecated `runJavascript` in favor of `runJavascriptForResult` in 2.2.0 (https://github.com/flutter/plugins/blob/master/packages/webview_flutter/webview_flutter/CHANGELOG.md#220), this change affects `google_maps_flutter_tizen` and `webview_flutter_tizen` in dart analyze.

@bwikbs @seungsoo47 
Could you update the package versions?

**Review by WonyoungChoi — APPROVED** [review]

File: analysis_options_plus.yaml [hunk 1]
```diff
@@ -1,17 +1,14 @@
...
     - '**/*.g.dart'
     - 'lib/src/generated/*.dart'
 
 linter:
   rules:
-    public_member_api_docs: true
     prefer_final_in_for_each: true
-    prefer_final_locals: true
\ No newline at end of file
+    prefer_final_locals: true
```
Comment (swift-kim) [inline analysis_options_plus.yaml:14]:
```suggestion
    prefer_final_locals
```
https://github.com/fluttercommunity/plus_plugins/blob/main/analysis_options.yaml

**Review by swift-kim — APPROVED** [review]

**Comment by bwikbs** [comment]
> Could you update the package versions?

Sorry for the late reply, I'll update right away!

File: analysis_options_plus.yaml (hunk 1 above)
Comment (HakkyuKim) [reply analysis_options_plus.yaml]: Done!

**Comment by HakkyuKim** [comment]
> > Could you update the package versions?
> 
> Sorry for the late reply, I'll update right away!

`dart analyze` now passes, thanks for the fix!

**Pull request merged**